test-requirements.txt

```diff
@@ -1,2 +1,4 @@
 pytest
 coverage
+twisted
```

Comment:
@lvh - Would you mind explaining how the link applies here? `twisted[tls]` is a dependency that in my mind should be set in `install_requires`, as this is a library.